Fix player hitbox blocking wiring tools raycast detection

AterialDawn opened the pull request:
Apply a 200-unit offset to raycast tests in the AWiremodBaseTool::GetTargetLookAt function

When moving around for long enough (potentially with a hypertube), the wiring tools will eventually have trouble detecting other entities besides the player entity. This fixes that by applying an (arbitrarily chosen) offset to the GetTargetLookAt raycast of 200 units. The raycast will skip the player bounding box, and correctly detect the intended look-at target

Another helpful user encountered the bug, and wrote a full description (with a video) showing the issue [on Discord](https://discord.com/channels/697414436322803763/1404985785203294389)

Scordium commented on Source/FicsitWiremod/Private/Equipment/WiremodBaseTool.cpp:
```diff
@@ -16,10 +16,11 @@ void AWiremodBaseTool::SetOutline(AActor* Object, EOutlineColor Color)
+	FVector End = camera->GetCameraLocation() + camera->GetActorForwardVector() * (TraceDistance + RaycastStartOffset);
 	FHitResult Result;
 
 	Success = UKismetSystemLibrary::LineTraceSingle(this, Start, End, Channel, false, TArray<AActor*>(), EDrawDebugTrace::None, Result, true);
```
While the offset works, it's incredibly fragile and hard to configure properly.
Kismet has a parameter to filter out unwanted actors from the raycast
```cpp
TArray<AActor*> RaycastIgnoredActors =
{
	GetInstigatorCharacter() //Ignoring player character holding the tool (self-collission)
};
	
Success = UKismetSystemLibrary::LineTraceSingle(this, Start, End, Channel, false, RaycastIgnoredActors, EDrawDebugTrace::None, Result, true);
```

AterialDawn commented:
Updated, tested, works!

Current example, with incorrect functionality
<img width="1024" height="576" alt="image" src="https://github.com/user-attachments/assets/4bbcdbf1-836f-4aa5-873b-10693ee26eba" />

Updated, and correct functionality!
<img width="1024" height="576" alt="image" src="https://github.com/user-attachments/assets/db069f79-4c0a-452c-b650-7a47ab600ba4" />

Scordium commented:
Tested and merged. Thanks.
